package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@izettle/javascript-env",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Any opinions on just bumping minor version? Only possible breaking changes are Webpack and webpack-dev-server dependencies which are bumped from 2.1 to 2.2. Ping @jalet